### PR TITLE
docs(component-store): update docs to add dictionnary of obs inside select

### LINKED ISSUE #3704: ComponentStore: Add documentation for using `ComponentStore.select` with observable dictionaries
### Information

Using `ComponentStore.select` with observable dictionaries is introduced in [this PR](https://github.com/ngrx/platform/pull/3629). This feature should be also added to [ComponentStore documentation](https://ngrx.io/guide/component-store/read#combining-selectors).

Read more about this feature [here](https://dev.to/ngrx/announcing-ngrx-v15-standalone-apis-type-safe-projectors-component-and-componentstore-updates-and-more-l7#simplified-view-models-with-componentstore).

### Documentation page

https://ngrx.io/guide/component-store/read#combining-selectors

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by tomalaforge**
add explanation about the new signature of select to accept a dictionnary of obs to create a view model

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #3704 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 1]
```diff
@@ -67,6 +67,19 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
 }
 </code-example>
 
+## Creating a View Model
+
+Creating view models is a recommended way to consolidate multiple streams in a clean API to provide to your component template.
+The `select` method accepts a dictionnary of observables as input and return an observable of dictionnary as output. View models can be written the following way:
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/component-store/read.md:73]:
```suggestion
The `select` method accepts a dictionary of observables as input and returns an observable of the dictionary of values as output. View models can be written the following way: 
```

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 2]
```diff
@@ -67,6 +67,19 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
...
+
+Creating view models is a recommended way to consolidate multiple streams in a clean API to provide to your component template.
+The `select` method accepts a dictionnary of observables as input and return an observable of dictionnary as output. View models can be written the following way: 
+
+<code-example header="movies.store.ts">
+  private readonly vm$ = this.select({
+    movies: movies$,
+    userPreferredMovieIds: userPreferredMovieIds$,
+    userPreferredMovies: userPreferredMovies$
+  });
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/component-store/read.md:80]:
```suggestion
  private readonly vm$ = this.select({
    movies: this.movies$,
    userPreferredMovieIds: this.userPreferredMovieIds$,
    userPreferredMovies: this.userPreferredMovies$
  });
```

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 3]
```diff
@@ -104,11 +117,10 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
...
 
-  private readonly fetchMoviesData$ = this.select(
-    moviesPerPage$,
-    currentPageIndex$,
-    (moviesPerPage, currentPageIndex) => ({moviesPerPage, currentPageIndex}),
-    {debounce: true}, // 👈 setting this selector to debounce
+  private readonly fetchMoviesData$ = this.select({
+    moviesPerPage: moviesPerPage$,
+    currentPageIndex: currentPageIndex$
+  },{debounce: true}, // 👈 setting this selector to debounce
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/component-store/read.md:123]:
```suggestion
  private readonly fetchMoviesData$ = this.select({
    moviesPerPage: this.moviesPerPage$,
    currentPageIndex: this.currentPageIndex$
  },{debounce: true}, // 👈 setting this selector to debounce
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: projects/ngrx.io/content/guide/component-store/read.md (hunk 2 above)
Comment (tomalaforge) [reply projects/ngrx.io/content/guide/component-store/read.md:80]: well seen

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 4]
```diff
@@ -67,6 +67,19 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
 }
 </code-example>
 
+## Creating a View Model
+
+Creating view models is a recommended way to consolidate multiple streams in a clean API to provide to your component template.
+The `select` method accepts a dictionary of observables as input and returns an observable of the dictionary of values as output. View models can be written the following way:
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/read.md:73]:
```suggestion
The `select` method accepts a dictionary of observables as input and returns an observable of the dictionary of values as output. View models can be written in the following way: 
```

**Review by timdeschryver — APPROVED** [review]
Thanks @tomalaforge

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**